ecoscope/analysis/classifier.py

```diff
@@ -75,7 +85,19 @@ def apply_classification(
     classifier = classifier_class(dataframe[input_column_name].to_numpy(), **kwargs)
     if labels is None:
         labels = classifier.bins
+
+        # Generate range labels if our bins are numeric
+        if np.issubdtype(dataframe[input_column_name].dtype, np.number) and label_ranges:
+            # We could do this using mapclassify.get_legend_classes, but this generates a cleaner labely
+            ranges = [f"0 - {labels[0]:.{label_decimals}f}"]
```

Comment:
One concern here is if the min value of the classified value is not zero? E.g., if we classify NDVI values into bins but their value might run from -0.85 to 0.95 for example.

Reply:
I can remove this so it will always be `dataframe[input_column_name].min()`

Reply:
Done - have replaced 0 with `dataframe[input_column_name].min()`